### PR TITLE
[WIP] Remove Dependence on Swiftz

### LINKED ISSUE #8: Proxy Clashes with Swiftz
It really isn't a pressing problem because I can't think of many situations you could feasibly use `Proxy<X, X, X, X, Proxy<X>>`, but it's still something.

### PR DESCRIPTION
**Pull request by CodaFi**
Kinda sorta fixes #8 and assuages our fears about some duplications.

### CODE REVIEW

**Comment by CodaFi** [comment]
OK, the last thing left is `Either`.  It wouldn't be hard to ship our own and completely break dependence on Swiftz - call it `Vee` or `Disjunct` or `Sum` or even `Or`.

**Comment by CodaFi** [comment]
We also lose a couple of Monoid things in Basic, but those can mostly be recovered through `IntegerType` constraints

**Comment by pthariensflame** [comment]
Sorry for being unreachable for a while!  My long-planned upgrade to Focus also needs an Either, but doesn't really depend on Swiftx for anything else.  Should we maybe consider making a microframework just containing a minimal Either type?

File: .gitmodules [hunk 1]
```diff
@@ -7,3 +7,6 @@
 [submodule "Carthage/Checkouts/Swiftz"]
 	path = Carthage/Checkouts/Swiftz
 	url = https://github.com/typelift/Swiftz.git
+[submodule "Carthage/Checkouts/Operadics"]
```
Comment (pthariensflame) [inline .gitmodules:7]: You still need to remove Swiftz's submodule.

File: Aquifer/Simple.swift [hunk 2]
```diff
@@ -110,8 +108,8 @@ public func <-< <UO, UI, DI, DO, DDI, DDO, FR>(p : Proxy<DI, DO, DDI, DDO, FR>,
 
 private func eachRepr<UO, UI, G : GeneratorType>(var gen : G) -> ProxyRepr<UO, UI, (), G.Element, ()> {
 	if let v = gen.next() {
-		return ProxyRepr.Respond(const(v)) { _ in eachRepr(gen) }
+		return ProxyRepr.Respond({ v }) { _ in eachRepr(gen) }
```
Comment (pthariensflame) [inline Aquifer/Simple.swift]: This should be `{ _ in v }`, and the same applies below with `{ _ in () }`.

**Comment by CodaFi** [comment]
I wish the STL would just export an Either :(

**Comment by pthariensflame** [comment]
Me too.  There must be at least a few dozen radars about it now, right?

**Comment by CodaFi** [comment]
In light of recent events, I'm abandoning this effort.  Hopefully `@exported` stabilizes or Swift gets an `Either`.

**Pull request closed without merging**